pkg/deployments/zeroscaler/zeroscaler.go

```diff
@@ -197,6 +174,50 @@ func (z *zeroscaler) ensureNoMetricsCollection(deployment *appsv1.Deployment) {
 	}
 }
 
+func (z *zeroscaler) getMetricsCheckInterval(
+	deployment *appsv1.Deployment,
+) time.Duration {
+	metricsCheckInterval, err := k8s.GetMetricsCheckInterval(
```

Comment:
This is the only place in all of Osiris where `k8s.GetMetricsCheckInterval` is used. I think its logic should just move here, otherwise, it becomes a tiny bit confusing why in this package is calling an unexported `getMetricsCheckInterval` function that doesn't do too much more than turn around and call `k8s.GetMetricsCheckInterval`, which isn't useful anywhere but here. Consolidating this logic will just avoid some head scratching in the future.

Reply:
ok, I moved everything in the zeroscaler pkg, except for the annotation name const. I'm thinking about keeping all annotations names as const in the kubernetes pkg, instead of spreading them all over the code base. Tell me what you think.

Reply:
I think my preference is to, initially, keep constants closer to where they are used. I am even a fan lately of defining constants right in functions. Once it needs to be used across more than one function, it gets promoted to the package level. Once it needs to be used across more than one package, it gets moved to a third/supporting package (such as the `kubernetes` package you referenced).